include/seqan3/alphabet/gap/gapped.hpp

```diff
@@ -83,58 +83,26 @@ struct gapped_alphabet : public union_composition<alphabet_t, gap>
     using typename union_composition<alphabet_t, gap>::rank_type;
     using typename union_composition<alphabet_t, gap>::char_type;
 
-    /*!\brief Returns true if it is a gap
-     * \details
-     * ```cpp
-     * gapped_alphabet<dna4> letter = dna4::T;
-     *
-     * if (!letter.is_gap())
-     *     std::cout << "T is NOT a gap character";
-     *
-     * letter.set_gap();
-     * if (letter.is_gap())
-     *     std::cout << "Now it is a gap character";
-     * ```
-     */
-    constexpr bool is_gap() const
-    {
-        return _value == value_size - 1;
-    }
-
-    /*!\brief Change it into a gap.
-     * \details
-     * ```cpp
-     * gapped_alphabet<dna4> letter;
-     * letter.set_gap();
-     *
-     * // the same as set_gap()
-     * letter = gap::GAP;
-     * ```
-     */
-    constexpr gapped_alphabet set_gap()
+    //!\copydoc union_composition::assign_char
+    constexpr gapped & assign_char(char_type const c) noexcept
     {
-        _value = value_size - 1;
+        union_composition<alphabet_t, gap>::assign_char(c);
```

Comment:
Nop, you can't because the return type would than be `union_composition`, that would violate the `alphabet_concept`, because it requires `gapped<>` as return type.